pkg/uploader/uploader.go

```diff
@@ -62,7 +62,7 @@ func (up *uploader) Upload(rootDir string, dstURL string) error {
 	if err != nil {
 		return fmt.Errorf("invalid destination AMS URL ('%s'): %w", dstURL, err)
 	}
-	u.Path = path.Join(u.Path, "/sap/ams/v1/bundles/SAP.tar.gz")
+	u.Path = path.Join(u.Path, fmt.Sprintf("/sap/ams/v1/dcl/base-upload/%s", amsInstanceID))
```

Comment:
```suggestion
	u.Path = path.Join(u.Path, "/sap/ams/v1/dcl/base-upload/", amsInstanceID))
```

Reply:
also: shouldn't this use the "new" path:
```suggestion
	u.Path = path.Join(u.Path, "/sap/ams/v1/ams-instances/", amsInstanceID, "dcl-upload")
```